### ISSUE TITLE
mask in reconstruction loss

### ISSUE THREAD

**Issue by jychoi118** [rosinality/semantic-pyramid-pytorch#2]
I think your reconstruction loss is missing mask, which should be applied after subtracting two features as in (3) of the paper.
![image](https://user-images.githubusercontent.com/36615789/83992749-48326800-a98c-11ea-9fe9-140ed1c96e61.png)

https://github.com/rosinality/semantic-pyramid-pytorch/blob/6d4796cbb2abdbe9afaafb8dae75b60a3b619d2d/train.py#L146

**Comment by rosinality** [comment]
Oh, thank you! Fixed it in d3777bf.
